internals/approval_defs.py

```diff
@@ -70,7 +70,9 @@ def fetch_owners(url):
     logging.error('Got response %r', response)
     raise ValueError('Could not get OWNERS file')
 
-  for line in response.iter_lines():
+  decoded = base64.b64decode(response.content)
+  for line in decoded.split('\n'):
```

Comment:
Does it have to be python3 compatible? in Python 3 `base64.b64decode` return bytesstring and it splits by `(b'\n')`, IIUC.

Reply:
Good catch.  I added a .decode() to convert to unicode.

I also fixed another case of the same py3 incompatibility in the fetchmetrics.py file.  And, in adding unit tests for that I realized that some existing unit tests needed more mocks to avoid actually hitting the OWNERS file (which make those tests flakey).